### ISSUE TITLE
Failed to Import Metadata Exported from v1.3.1 

### ISSUE THREAD

**Issue by zprobs** [hasura/graphql-engine#7217]
After exporting metadata as JSON from our v1.3.1 project via the console, we get the following error after trying to import it into a new v2.0.1-cloud.1 project:

`Failed importing metadata key "args" not found`

**Comment by ro-savage** [comment]
We tried to export from hosted v2.0.1 to Cloud v2.0.1 and get the error `key table not found`

![image](https://user-images.githubusercontent.com/9244507/125394504-e1d51000-e3fd-11eb-8eef-b102546cf8d3.png)

When we tried v1.3.1 we got the `args` error.

It appears impossible to export/import with v2 at the moment.

@0x777

**Comment by sordina** [comment]
Hi @zprobs,

We've made some parsing changes recently that we believe may address the issue you have been experiencing. Would you mind testing on version 2.0.3 to see if the problem still persists?

**Comment by zprobs** [comment]
> Hi @zprobs,
> 
> We've made some parsing changes recently that we believe may address the issue you have been experiencing. Would you mind testing on version 2.0.3 to see if the problem still persists?

Hi @sordina, I just tested this again with version 2.0.4 and the issue is still present. Let me know if there is anything I can do to help such as providing the metadata file. If there are no known workarounds to this problem I will continue using v1.3.1 for the time being.

**Comment by b3tts32** [comment]
Getting same issue on v2.0.5 when importing metadata from v1.3.2

**Comment by tirumaraiselvan** [comment]
@zprobs @b3tts32 If you could paste the 1.3 metadata, it will help a lot! If it's sensitive data, you could also email to `tiru@hasura.io` (feel free to redact/de-identify values)

**Comment by b3tts32** [comment]
Sent via email

**Comment by sordina** [comment]
Hi @zprobs, @b3tts32 could you show the curl command for the API call you're using to import the metadata?

**Comment by tirumaraiselvan** [comment]
@b3tts32 Could you help us with the questions above if you are still facing this error?

**Comment by tfriskeAltran** [comment]
@tirumaraiselvan , I have encountered the same issue trying to import using the v2.0.4 console with the following simple metadata:

{
  "version": 2,
  "tables": [],
  "remote_schemas": [
    {
      "name": "accountmanager",
      "definition": {
        "url": "http://account-manager-app:8080/graphql/",
        "timeout_seconds": 60,
        "forward_client_headers": true
      }
    }
  ]
}

**Comment by PNAtitles** [comment]
I have the **same issue** in **Hasura Cloud**:
Created a **new project** in **cloud** (**v2.0.3-cloud.1**) and tried to **import** the **meta-data** from **another cloud project** (**v1.3.3-cloud.9**). 
The import fails with the error:
> "Failed importing metadata key "args" not found"

.

**Comment by stayingcool** [comment]
I have the same issue in Hasura Cloud:
Created a new project in the cloud (v2.0.3-cloud.1) and tried to import the meta-data from another cloud project (v1.3.3-cloud.9).
The import fails with the error:

"Failed importing metadata key "args" not found"

**Comment by tirumaraiselvan** [comment]
Yes, this seems to happen when importing via console. We will fix this ASAP. Meanwhile, you can use the `replace_metadata` API call to apply your v2 metadata: https://hasura.io/docs/latest/graphql/core/api-reference/schema-metadata-api/manage-metadata.html
